<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Save</class>
 <widget class="QDialog" name="Save">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Wpisz swoje inicja≈Çy (3 znaki)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="nazwaTxt"/>
      </item>
      <item>
       <widget class="QLabel" name="aktWynik">
        <property name="text">
         <string>Aktualny wynik=</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ZapiszBtn">
        <property name="text">
         <string>Zapisz</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="ListaWynik"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
